clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -931,7 +929,7 @@ public void testMissingOffsetNoResetPolicy(GroupProtocol groupProtocol) {
 
         // lookup committed offset and find nothing
         client.prepareResponseFrom(offsetResponse(Collections.singletonMap(tp0, -1L), Errors.NONE), coordinator);
-        assertThrows(NoOffsetForPartitionException.class, () -> consumer.poll(Duration.ZERO));
+        assertThrows(NoOffsetForPartitionException.class, () -> consumer.poll(Duration.ofMillis(1)));
```

Comment:
Is "1 millis" stable if the AsyncConsumer depends on background thread to complete the request?

Reply:
It is stable on my local but not sure in CI.

Reply:
maybe we can increase it to 3 seconds at least? It does not burn out our CI

Reply:
Done.